packages/mobile/src/components/usdc-manual-transfer/USDCManualTransfer.tsx

```diff
@@ -121,61 +65,41 @@ export const USDCManualTransfer = ({
     trackEvent(make(analytics))
   }, [USDCUserBank, analytics, toast])
 
-  const handleLearnMorePress = useCallback(() => {
-    onPressLearnMore()
-  }, [onPressLearnMore])
-
   return (
-    <View style={styles.root}>
-      <Text style={styles.explainer}>{messages.explainer}</Text>
-      <View style={styles.qr}>
-        {USDCUserBank ? (
-          <QRCode size={160} style={styles.qr} value={USDCUserBank} />
-        ) : null}
-      </View>
-      <AddressTile
-        title={messages.usdcBalance}
-        address={USDCUserBank ?? ''}
-        left={<LogoUSDC height={spacing(6)} />}
-        analytics={analytics}
-        balance={USDC(balanceBN ?? new BN(0)).toLocaleString('en-US', {
-          roundingMode: 'floor',
-          minimumFractionDigits: 2,
-          maximumFractionDigits: 2
-        })}
-      />
-      <View style={styles.disclaimerContainer}>
-        <IconError
-          width={spacing(6)}
-          height={spacing(6)}
-          fill={neutral}
-          style={styles.icon}
-        />
-        <View style={styles.hintContainer}>
-          <View style={styles.shrink}>
-            <Text style={styles.disclaimer}>{messages.hint}</Text>
-          </View>
-          <Text
-            style={styles.learnMore}
-            color='primary'
-            onPress={handleLearnMorePress}
-          >
-            {messages.learnMore}
+    <Flex ph='l' gap='xl'>
+      <CashBalanceSection />
+      <Flex row gap='l' alignItems='center'>
+        {USDCUserBank ? <QRCode size={160} value={USDCUserBank} /> : null}
+        <Flex flex={1}>
+          <Text size='l'>{messages.explainer}</Text>
+        </Flex>
+      </Flex>
+      <AddressTile address={USDCUserBank} analytics={analytics} />
+      <Flex
+        row
+        gap='l'
+        alignItems='center'
+        ph='l'
+        pv='m'
+        backgroundColor='surface2'
+        border='strong'
+        borderRadius='m'
+      >
+        <IconError width={spacing(6)} height={spacing(6)} fill={neutral} />
```

Comment:
size='l' / the other various sizes for all the icons in this pr plz, color='default' too.